### DISCUSSION TITLE
RTE Disappears After Clicking Formatting Button in GrapesJS React with Tailwind CSS

### DISCUSSION THREAD

**Discussion by fredrickreuben** [GrapesJS/grapesjs#6014] [General]
I am using the GrapesJS React version with custom CSS styles using Tailwind CSS. When the Rich Text Editor (RTE) is enabled, it initially works as expected. However, when I click a formatting button (such as the bold button), the selected text is correctly formatted, but the entire RTE disappears immediately. This forces me to re-enable the RTE to continue editing.

Steps to Reproduce:

Enable the RTE on a text component.
Select some text.
Click a formatting button like bold.
Expected Behavior:
The RTE should remain active and visible after clicking a formatting button.

Actual Behavior:
The RTE disappears immediately after clicking a formatting button.

Additional Information:

GrapesJS version: [Specify version]
React version: [Specify version]
Tailwind CSS version: [Specify version]
It appears to be an issue with the event handling or state management of the RTE. Any guidance or fixes for this problem would be greatly appreciated. Thank you!

**Comment by artf** [comment]
Any reproducible demo?
